charts/ratify/templates/provider.yaml

```diff
@@ -11,3 +11,19 @@ spec:
   {{- if .Values.provider.auth }}
   caBundle: {{ required "You must provide .Values.provider.tls.cabundle when .Values.provider.auth is set" .Values.provider.tls.cabundle }}
   {{- end }}
+
+---
+
+apiVersion: externaldata.gatekeeper.sh/v1alpha1
+kind: Provider
+metadata:
+  name: ratify-mutation-provider
+spec:
+  url: {{ if .Values.provider.auth }}https{{ else }}http{{ end }}://{{ include "ratify.fullname" .}}.{{ .Release.Namespace }}:6001/ratify/gatekeeper/v1/mutate
+  timeout: 1
+  {{- if .Values.provider.tls.skipVerify }} # allow gatekeeper with version < 3.9.x
+  insecureTLSSkipVerify: true # enable this if the provider uses HTTP so that Gatekeeper can skip TLS verification.
```

Comment:
afaik, this is no longer supported with `v3.11`. @sozercan to confirm!

Reply:
will remove. waiting for gatekeeper version PR to merge and then will rebase